clang-tools-extra/clangd/ClangdLSPServer.cpp

```diff
@@ -817,7 +817,7 @@ void ClangdLSPServer::onCodeAction(const CodeActionParams &Params,
   std::vector<Command> CCommands;
   // Convert the diagnostics into 3C commands.
   for (const Diagnostic &D : Params.context.diagnostics) {
-    AsCCCommands(D, CCommands);
+    As3CCommands(D, CCommands);
```

Comment:
If you don't feel unhappy about doing it, how bout changing just this local variable from `CCommands` to `Commands` ?

Reply:
Sure.